libsolidity/codegen/ir/IRGeneratorForStatements.h

```diff
@@ -86,7 +86,10 @@ class IRGeneratorForStatements: public IRGeneratorForStatementsBase
 	IRVariable evaluateExpression(Expression const& _expression, Type const& _to);
 
 	/// Defines @a _var using the value of @a _value while performing type conversions, if required.
-	void define(IRVariable const& _var, IRVariable const& _value) { declareAssign(_var, _value, true); }
+	void define(IRVariable const& _var, IRVariable const& _value, bool _forceCleanup = false)
```

Comment:
One last thing ;-). It might be nice to document ``_forceCleanup`` in the comment here (and also for ``convert``), similar to the comment at ``expressionAsType``.